packages/components/molecules/f-searchbox/src/components/formElements/FormSearchField.vue

```diff
@@ -348,28 +348,4 @@ export default {
         background-color: $grey--offWhite;
     }
 }
-
-// Hide only visually, but have it available for screenreaders: h5bp.com/v
-.is-visuallyHidden {
-    border: 0;
-    clip: rect(0 0 0 0);
-    height: 1px;
-    margin: -1px;
-    overflow: hidden;
-    padding: 0;
-    position: absolute;
-    width: 1px;
-
-    // Extends the .is-visuallyhidden class to allow the element to be focusable when navigated to via the keyboard: h5bp.com/p
```

Comment:
Has this been ported out to Fozzie?

Reply:
I think it has been copied from fozzie: https://github.com/justeat/fozzie/blob/a1e5f92a0a8f3225660fbfdf64de2e2b5b6bbf7d/src/scss/trumps/_utilities.scss#L69

Reply:
Ah, yes, sorry I missed that.